src/Catalyst.Core.Lib.Tests/UnitTests/Validators/TransactionValidatorTests.cs

```diff
@@ -0,0 +1,155 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using Catalyst.Abstractions.Cryptography;
+using Catalyst.Core.Lib.Extensions;
+using Catalyst.Core.Lib.Validators;
+using Catalyst.Protocol.Cryptography;
+using Catalyst.Protocol.Network;
+using Catalyst.Protocol.Transaction;
+using Catalyst.Protocol.Wire;
+using FluentAssertions;
+using Google.Protobuf;
+using NSubstitute;
+using Serilog;
+using Xunit;
+
+namespace Catalyst.Core.Lib.Tests.UnitTests.Validators
+{
+    public sealed class TransactionValidatorTests
+    {
+        [Fact]
+        public void TransactionValidator_ValidateTransactionSignature_returns_false_when_signature_is_null()
+        {
+            var subbedLogger = Substitute.For<ILogger>();
+            var subbedContext = Substitute.For<ICryptoContext>();
```

Comment:
Should use FfiWrapper instead of Substitute CryptoContext as the sign/verify substitutes are just returning true or false.